.editorconfig

```diff
@@ -3,7 +3,7 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = false
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Wouldn't we need to reformat all code to conform?

Reply:
I had done this in https://github.com/google/ground-android/pull/2671/files which is causing all files to change indentation. So, this is a rollback for this file only